SAP/spartacus
# doc: CXSPA-8104 Adjust docs with steps required for migration when Angular 17.3.8 is installed

This PR contains adjustments in migration docs to cover issues that occurred when migrating the SPA app to 2211.x with Angular 17.3.8.
There are two new dependency issues that needs to be resolved by installing and updating some pakages manually:
- The following error has been resolved by the manual installation of `@angular-devkit/core` in `devDependencies`:
![Screenshot 2024-08-27 at 11 20 28](https://github.com/user-attachments/assets/105ebe17-331c-4a91-82a5-9d78eee039f1)
- After this, when the next error occurred, it was resolved by the manual upgrade of `rxjs` library to `v7.8.1`:
![Screenshot 2024-08-27 at 11 22 19](https://github.com/user-attachments/assets/2686292f-52c7-4b7a-919f-9df880c5bebc)

Steps to QA:
- prepare fresh app with SPA v6.8.0
- go through the migration steps from updated `2211-migration.md` file
- verify whether the process was smooth, without errors

Closes[ CXSPA-8104](https://jira.tools.sap/browse/CXSPA-8104)
